extensions/amp-social-share/1.0/social-share.js

```diff
@@ -70,48 +83,58 @@ export function SocialShare(props) {
  *   baseEndpoint: ?string,
  *   checkedWidth: ?number,
  *   checkedHeight: ?number,
+ *   params: !JsonObject,
  * }}
  */
 function checkProps(props) {
   const {
     'type': type,
-    'shareEndpoint': shareEndpoint,
-    'params': params,
-    'bindings': bindings,
+    'endpoint': endpoint,
+    'target': target,
     'width': width,
     'height': height,
+    // eslint-disable-next-line no-unused-vars
+    'tabIndex': tabIndex,
+    // eslint-disable-next-line no-unused-vars
+    'style': style,
+    // eslint-disable-next-line no-unused-vars
+    'ignoreParams': ignoreParams,
+    // eslint-disable-next-line local/no-rest
+    ...params
   } = props;
 
   // Verify type is provided
   if (type === undefined) {
     throw new Error(`The type attribute is required. ${NAME}`);
   }
 
-  // bindings and params props must be objects
-  if (params && !isObject(params)) {
-    throw new Error(`The params property should be an object. ${NAME}`);
-  }
-  if (bindings && !isObject(bindings)) {
-    throw new Error(`The bindings property should be an object. ${NAME}`);
-  }
-
-  // User must provide shareEndpoint if they choose a type that is not
+  // User must provide endpoint if they choose a type that is not
```

Comment:
"type that is not" what?

Reply:
"not pre-configured" (it's on the next line and was not changed)